modules/kvformat/linux-audit-scanner.c

```diff
@@ -134,9 +134,9 @@ _parse_linux_audit_style_hexdump(KVScanner *self)
 }
 
 KVScanner *
-linux_audit_scanner_new(void)
+linux_audit_scanner_new(KVScanner *base)
 {
```

Comment:
Hmmm.. I'm sure that we've talked about this... this is not a kind of  `_new` function (this is confusing, at least for me). I'll check other changes on tomorrow.